Core/src/main/java/com/plotsquared/core/plot/PlotArea.java

```diff
@@ -370,6 +373,40 @@ public void loadDefaultConfiguration(ConfigurationSection config) {
         this.spawnEggs = config.getBoolean("event.spawn.egg");
         this.spawnCustom = config.getBoolean("event.spawn.custom");
         this.spawnBreeding = config.getBoolean("event.spawn.breeding");
+
+        List<String> roadflags = config.getStringList("flags.default");
+        if (roadflags.isEmpty()) {
+            roadflags = config.getStringList("road.flags");
+            if (roadflags.isEmpty()) {
+                roadflags = new ArrayList<>();
+                ConfigurationSection section = config.getConfigurationSection("road.flags");
+                Set<String> keys = section.getKeys(false);
+                for (String key : keys) {
+                    if (!"default".equals(key)) {
+                        roadflags.add(key + ';' + section.get(key));
+                    }
+                }
+            }
+        }
+        this.getRoadFlagContainer().addAll(parseFlags(roadflags));
+
+        StringBuilder roadFlagBuilder = new StringBuilder();
+        Collection<PlotFlag<?, ?>> roadFlagCollection = this.getFlagContainer().getFlagMap().values();
+        if (roadFlagCollection.isEmpty()) {
+            roadFlagBuilder.append(Captions.NONE.getTranslated());
+        } else {
+            roadFlags = true;
+            String prefix = " ";
+            for (final PlotFlag<?, ?> flag : roadFlagCollection) {
+                Object value = flag.toString();
+                roadFlagBuilder.append(prefix).append(CaptionUtility
+                    .format(null, Captions.PLOT_FLAG_LIST.getTranslated(), flag.getName(),
+                        CaptionUtility.formatRaw(null, value.toString(), "")));
+                prefix = ", ";
+            }
+        }
+        PlotSquared.log(Captions.PREFIX + "&3 - road flags: &7" + roadFlagBuilder.toString());
```

Comment:
This could be changed to debug.

Reply:
It's log for the plot flags as well, so figured I'd keep the same.

Reply:
Ah ok, makes sense. I shifted a few things like that to debug in the past so it doesn't clog up things unnecessary when you have lots of worlds.
I've seen people having like 50+ worlds and that can get rather spammy then.